CONTRACT.md

```diff
@@ -200,31 +200,20 @@ loadAndUpdateRemote output is a full Cumulus Message as a json blob.
 
 ### `loadNestedEvent` input
 
-* `<event_json>` to cumulus-message-adapter `loadNestedEvent` should be a full Cumulus Message as defined above.
+* `event` should be a full Cumulus Message as defined above, with optional `task_config` section defined.
 
-* `<context_json>` to cumulus-message-adapter `loadNestedEvent` should be the context from the lambda.
+### `loadNestedEvent` Details
 
-* `<schemas_json>` to cumulus-message-adapter `loadNestedEvent` should be an object with filepaths to json schemas for the `input` and `config` properties that a "business function" expects to receive
+`loadNestedEvent` will read the configuration specified in `task_config`, and return a response object with:
 
-**`loadNestedEvent` Details:**
```

Comment:
For the most part this was already incorrect, the update is intended to condense/simplify.